<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="George-PC" name="Default test" tests="2" failures="0" timestamp="19 May 2013 10:29:18 GMT" time="9.847" errors="0">
  <testcase name="@BeforeClass createMailUser" time="0.0" classname="com.example.tests.SignupTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.example.fw.JamesHelper.initTelnetSession(JamesHelper.java:21)
	at com.example.fw.JamesHelper.doesUserExist(JamesHelper.java:88)
	at com.example.tests.SignupTest.createMailUser(SignupTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- @BeforeClass createMailUser -->
  <testcase name="@AfterClass deleteMailUser" time="0.0" classname="com.example.tests.SignupTest">
    <skipped/>
  </testcase> <!-- @AfterClass deleteMailUser -->
  <testcase name="@AfterTest tearDown" time="0.0" classname="com.example.tests.SignupTest">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="newUserShouldSignUp" time="0.0" classname="com.example.tests.SignupTest">
    <skipped/>
  </testcase> <!-- newUserShouldSignUp -->
  <testcase name="registeredUserShouldNotSignUp" time="0.0" classname="com.example.tests.SignupTest">
    <skipped/>
  </testcase> <!-- registeredUserShouldNotSignUp -->
</testsuite> <!-- Default test -->
